XRTK-Core/Packages/com.xrtk.core/Services/TeleportSystem/MixedRealityTeleportSystem.cs

```diff
@@ -220,7 +220,7 @@ private void ProcessTeleportationRequest(TeleportEventData eventData)
         {
             isProcessingTeleportRequest = true;
 
-            var cameraParent = MixedRealityToolkit.Instance.MixedRealityPlayspace;
+            var cameraParent = MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform;
```

Comment:
@SimonDarksideJ This is the only change to the teleport system.

Maybe we need to update this to be the playspace

Reply:
I'll go back to the development branch and compare in the morning, Will do a couple of GIF's and see what's going on

Reply:
Were you getting the same behavior on the dev branch?

Reply:
Tested against release and dev and found this change is not causing the teleportation issue.  Raised #390